index.html

```diff
@@ -332,23 +437,193 @@ <h4>SvgRenderingTemplate</h4>
         </p>
+
+        <section>
+          <h3>The `svg-mustache` Render Suite</h3>
+          <p>
+The `svg-mustache` render suite uses the Mustache templating language to
+modify an SVG file, which is then used to render a visual representation
+of the [=verifiable credential=].
+          </p>
+
+          <p>
+In the example below, a fully embedded SVG file is used as the rendering
+template.
+          </p>
+
+          <pre class="example nohighlight"
+               title="Basic usage of the svg-mustache render suite">
+{
+  <span class="comment">...</span>
+  "renderMethod": {
+    "type": "TemplateRenderMethod",
+    "renderSuite": "svg-mustache",
+    <span class="comment">// the SVG file is embedded in the VC</span>
+    "template": "data:image/svg;base64,Qjei89...3jZpW"
+  }
+}
+          </pre>
+
+          <p>
+The next example links to the SVG file on the Web and secures it against
+modification by using the `digestMultibase` property.
+          </p>
+
+          <pre class="example nohighlight"
+              title="A remotely hosted SVG file for an SVG render template">
+{
+<span class="comment">...</span>
+"renderMethod": {
+  "type": "TemplateRenderMethod",
+  "renderSuite": "svg-mustache",
+  "template": {
+    <span class="comment">// this SVG file is fetched from the Web</span>
+    "id": "https://degree.example/bachelors.svg",
+    "mediaType": "image/svg",
+    "digestMultibase": "zQmerWC85Wg6wFl9znFCwYxApG270iEu5h6JqWAPdhyxz2dR"
+  }
+}
+          </pre>
+
+          <p>
+The next example links to the rendering template on the Web and secures it
+using the `digestMultibase` property:
+          </p>
+
+          <pre class="example nohighlight"
+              title="A remotely hosted SVG render method">
+{
+<span class="comment">...</span>
+"renderMethod": {
+  <span class="comment">// this render method is fetched from the Web</span>
+  "id": "https://degrees.example/bachelors-svg.jsonld",
+  "type": "TemplateRenderMethod",
+  "renderSuite": "svg-mustache",
+  "digestMultibase": "zQmG270iEu5h6JqWAPdhyxz2dRerWC85Wg6wFl9znFCwYxAp"
+}
+          </pre>
+
+        </section>
+
+        <section>
+          <h3>The `pdf-mustache` Render Suite</h3>
+          <p>
+The `pdf-mustache` render suite uses the Mustache templating language to
+modify a PDF file, which is then used to render a visual representation
+of the [=verifiable credential=].
+          </p>
+
+          <p>
+In the example below, a fully embedded PDF file is used as the rendering
+template.
+          </p>
+
+          <pre class="example nohighlight"
+               title="Basic usage of the pdf-mustache render suite">
+{
+  <span class="comment">...</span>
+  "renderMethod": {
+    "type": "TemplateRenderMethod",
+    "renderSuite": "pdf-mustache",
+    <span class="comment">// this PDF file is embedded in the VC</span>
+    "template": "data:application/pdf;base64,k309SK...pwK83b"
```

Comment:
Do we have examples of a PDF file containing Mustache template code that survives a roundtrip through a Mustache template processor?

Reply:
Not yet, no... all we have is a standards architect from Adobe telling us that Mustache is a completely fine thing to use for a PDF file. That said, we should make sure this works in practice. :)